apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -764,6 +843,36 @@ static CodeableConcept createCodeableConcept(
     return concept;
   }
 
+  /**
+   * Adds a qualification {@link CodeableConcept} to the given careTeam component, if the input code
+   * optional is not empty. If the code is empty, returns with no effect. Can safely be called to
+   * add qualification only if the value is present.
+   *
+   * @param careTeam the care team to add the
+   * @param rootResource the root resource to use for the coding
+   * @param ccwVariable the ccw variable to use for the coding
+   * @param code an optional to create the {@link CodeableConcept} from; if empty, method returns
+   *     with no action taken
+   */
+  static void addCareTeamQualification(
+      CareTeamComponent careTeam,
+      IAnyResource rootResource,
+      CcwCodebookInterface ccwVariable,
+      Optional code) {
```

Comment:
This should at least be `Optional<?>` though an overload would be better.

Reply:
Fixed them to use this instead.